guides/v2.3/graphql/release-notes.md

```diff
@@ -15,6 +15,30 @@ These release notes can include:
 -   {:.new}New features
 -   {:.fix}Fixes and improvements
 
+## {{site.data.var.ce}} 2.3.2
+
+- {:.new} **Added mutations to support the following cart operations and checkout:**
+  - Add [simple products]({{page.baseurl}}/graphql/reference/quote-add-simple-products.html) and [virtual products]({{page.baseurl}}/graphql/reference/quote-add-simple-products.html) to a cart.
+  - Set the [shipping address]({{page.baseurl}}/graphql/reference/quote-set-shipping-address.html). Address books are supported.
+  - Set the [billing address]({{page.baseurl}}/graphql/reference/quote-set-billing-address.html). Address books are supported.
+  - Set the [shipping method]({{page.baseurl}}/graphql/reference/quote-shipping-method.html). Supported methods include DHL, FedEx, Flat Rate, Free Shipping, Table Rate, UPS, and USPS.
+  - Set the [payment method]({{page.baseurl}}/graphql/reference/quote-payment-method.html). Supported methods include Bank Transfer, Cash on Delivery, Check/Money Order, and Purchase Order.
+  - [Apply]({{page.baseurl}}/graphql/reference/quote-apply-coupon.html) or [remove]({{page.baseurl}}/graphql/reference/quote-remove-coupon.html) a coupon to a cart.
```

Comment:
Because "remove a coupon to a cart" sounds weird what about "Apply or remove cart coupons."?